SPECS/kernel/kernel.spec

```diff
@@ -513,6 +520,11 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %endif
 
 %changelog
+* Wed Aug 11 2021 Rachel Menge <rachelmenge@microsoft.com> - 5.10.57.1-1
```

Comment:
nit - change date to aug 12 to match other specs?

Reply:
fixed :)